tests/FsCheck.Test/Gen.fs

```diff
@@ -119,6 +119,12 @@ module Gen =
         Gen.listOfLength length (Gen.constant v)
         |> sample1
         |> ((=) (List.init length (fun _ -> v)))
+
+    [<Property>]
+    let Shuffle (NonEmptySet (xs:Set<int>)) =
+        Gen.shuffle xs
+        |> sample 10
+        |> List.exists ((<>) (Set.toSeq xs))
```

Comment:
>sequences don't have structural equality (neither would I expect them to)

Same here :confused:

I've observed the test failing in the beginning, by taking a list as input, then started changing the test - even though [I shouldn't do that](http://blog.ploeh.dk/2013/04/02/why-trust-tests/) - and then it eventually became less trustworthy.

Now the question is, how should we *verify* this beast...